cmd/argo/commands/list.go

```diff
@@ -27,6 +27,7 @@ type listFlags struct {
 	completed     bool
 	running       bool
 	resubmitted   bool
+	pending       bool
```

Comment:
Should add [a test](https://github.com/argoproj/argo-workflows/blob/28ffd1c442260dc8ef222062c41d2c33e880bd65/cmd/argo/commands/list_test.go#L43) for this as well